gmso/tests/test_convert_parmed.py

```diff
@@ -47,3 +47,67 @@ def test_from_parmed_parametrized_structure(self, angles):
         lengths = u.nm * [0.714, 0.7938, 0.6646]
         assert allclose(top.box.lengths, lengths)
         assert allclose(top.box.angles, angles)
+
+    def test_to_parmed_simple(self):
+        struc = pmd.load_file(get_fn('ethane.top'), xyz=get_fn('ethane.gro'))
+        top = from_parmed(struc)
+        struc_from_top = to_parmed(top, refer_type=False)
+
+        assert len(struc.atoms) == len(struc_from_top.atoms)
+        assert len(struc.bonds) == len(struc_from_top.bonds)
+        assert len(struc.angles) == len(struc_from_top.angles)
+        assert len(struc.dihedrals) == len(struc.dihedrals)
+
+    def test_to_parmed_full(self):
+        struc = pmd.load_file(get_fn('ethane.top'), xyz=get_fn('ethane.gro'))
+        top = from_parmed(struc)
+        struc_from_top = to_parmed(top)
+
+        #assert struc.atom_types == struc_from_top.atom_types
+        assert struc.bond_types == struc_from_top.bond_types
+        assert struc.angle_types == struc_from_top.angle_types
+        assert struc.dihedral_types == struc_from_top.dihedral_types
+        assert struc.rb_torsion_types == struc_from_top.rb_torsion_types
```

Comment:
How are pmd's equality overrides done? Does this go through and check each attribute of the object against the other? My first guess in a round-trip conversion would be to compare the GMSO objects but this could be easier (and I don't see any reason to believe it tests different behavior, unless there was a reason to do two full loops)

Reply:
For Parmed, two bond types are equal if their parameters are equal. Parmed Structure does not store atom_types, so I cannot compare that one